client/llb/source.go

```diff
@@ -452,6 +453,45 @@ func Differ(t DiffType, required bool) LocalOption {
 	})
 }
 
+func OCILayout(contentStoreID, digest string, opts ...OCILayoutOption) State {
```

Comment:
use type `digest.Digest`

Reply:
Yeah, that is a simple protection. Done.